### DISCUSSION TITLE
How to force spell checking based on token scope?

### DISCUSSION THREAD

**Discussion by koehlma** [streetsidesoftware/vscode-spell-checker#825] [Q&A]
Hey there!

Is there a way to force spell checking for different token scopes?

If I remember correctly, this was possible a few years ago. Concretely, I have the problem that for LaTeX spellchecking does not work in all macros so I would like to force spellchecking upon all `text.tex.latex` scopes.

In any case, thank you so much for this amazing piece of software.

**Comment by Jason3S** [comment]
The spell check does not currently support that, but the plan (depending upon funding) is to support that feature.
